internal/kibana/connector.go

```diff
@@ -198,6 +197,26 @@ func resourceConnectorRead(ctx context.Context, d *schema.ResourceData, meta int
 	return flattenActionConnector(connector, d)
 }
 
+func resourceConnectorsRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	client, diags := clients.NewApiClientFromSDKResource(d, meta)
+	if diags.HasError() {
+		return diags
+	}
+	name := d.Get("name").(string)
+	spaceId := d.Get("space_id").(string)
+
+	connector, diags := kibana.GetConnectorByName(ctx, client, name, spaceId)
+	if diags.HasError() {
+		return diags
+	}
+	id, diags := client.ID(ctx, connector.ConnectorID)
```

Comment:
This I'd should be the connector I'd and space I'd like were building on line 142.